GitDemo/Program.cs

```diff
@@ -1,12 +1,15 @@
 ﻿using System;
+using System.Runtime.CompilerServices;
 
+[assembly: InternalsVisibleTo("GitDemo.Tests")]
 namespace GitDemo
 {
-    class Program
+    internal class Program
     {
-        static void Main(string[] args)
-        {
-            Console.WriteLine("Hello Airsphere Team!");
-        }
+        private static void Main(string[] args)
+            => Console.WriteLine(GetMessage());
+
+        internal static string GetMessage()
+            => "Hello Airsphere Team!";
```

Comment:
Is ja alles falsch hier. So geht das aber gar nicht!
```suggestion
            => "Hello AirSphere Team!";
```